components/connectivity-adapter/internal/appregistry/service/graphql_request_builder.go

```diff
@@ -0,0 +1,28 @@
+package service
+
+import (
+	"fmt"
+
+	gcli "github.com/machinebox/graphql"
+)
+
+// TODO: Remove this file and migrate to GraphQLRequester
```

Comment:
why you are adding a new file with already deprecated type when GraphQLRequester exist?

Reply:
Unified